### PR TITLE
Provide a default gas price #33

### PR DESCRIPTION
**Pull request by rotcivegaf**
Provide a default gas price from ethgasstation.info

**Refe:** #33
**Branch:** Provide-a-default-gas-price

### CODE REVIEW

File: src/app/services/contracts.service.ts [hunk 1]
```diff
@@ -139,8 +141,17 @@ export class ContractsService {
       }) as Promise<number>;
     }
 
+    public async getGasPriceAvg(): Promise<number> {
+      return new Promise((resolve) => {
+        this.http.get('https://ethgasstation.info/json/ethgasAPI.json').subscribe((resp: any) => {
```
Comment (Agusx1211) [inline src/app/services/contracts.service.ts]: https://ethgasstation.info/json/ethgasAPI.json Should be an environment variable

File: src/app/services/contracts.service.ts [hunk 2]
```diff
@@ -139,8 +141,17 @@ export class ContractsService {
...
+          resolve(resp['average']);
```
Comment (Agusx1211) [inline src/app/services/contracts.service.ts]: We should provide a default value if ethgasstation is not working

**Comment by Agusx1211** [comment]
The gas price may be prefetched, it's unlikely to change each minute, and blocking operations by waiting for the response of a 3rd party API is risky.

Also, most of the Web3 providers are currently providing their gas price; maybe this is no longer required.

**Comment by rotcivegaf** [comment]
Now MetaMask provide a default gas price

**Pull request closed without merging**